utils.py

```diff
@@ -1,8 +1,47 @@
 from datetime import datetime, timedelta
+import pandas as pd
 
 DATETIME_FORMAT = "%m-%d-%Y"
 
 
+def file_to_list(file: str, column: str) -> list:
+    file_to_df = pd.read_csv(file, usecols=[column])
+    list_of_values = file_to_df[column].tolist()
+
+    if not list_of_values:  # no data in file
+        raise Exception("File is empty")
+
+    return list_of_values
+
+
+def state_parameter_validator(state_name: str) -> bool:
+    valid_states = file_to_list("tests/state_province.csv", "state")
+    is_valid_state = state_name in valid_states
+
+    if not is_valid_state:
+        raise Exception(
+            "{} is not valid. Use one of the following values:{}".format(
+                state_name, valid_states
+            )
+        )
+    else:
+        return
```

Comment:
this doesn't return a boolean, this just returns out of the function without returning anything